fix marcar como visto en kodi para bibliotecas con "special" en la ruta

SeiTaN opened the pull request.

superberny70 commented on python/main-classic/platformcode/library.py:
```diff
@@ -576,7 +576,12 @@ def mark_content_as_watched_on_kodi(item, value=1):
-                if d['file'].replace("/", "\\").endswith(item.strm_path.replace("/", "\\")):
+
+                filename = os.path.basename(item.strm_path)
+                head, tail = os.path.split(os.path.split(item.strm_path)[0])
```
Por q no usas filetools en lugar de os.path?

SeiTaN commented:
no caí :P, cambialo si quieres

2016-10-27 16:15 GMT+02:00 Superberny <notifications@github.com>:

> *@superberny70* commented on this pull request.
> ------------------------------
>
> In python/main-classic/platformcode/library.py
> <https://github.com/superberny70/pelisalacarta/pull/12#pullrequestreview-6067228>
> :
>
> > @@ -576,7 +576,12 @@ def mark_content_as_watched_on_kodi(item, value=1):
>          data = get_data(payload)
>          if 'result' in data:
>              for d in data['result']['movies']:
> -                if d['file'].replace("/", "\\").endswith(item.strm_path.replace("/", "\\")):
> +
> +                filename = os.path.basename(item.strm_path)
> +                head, tail = os.path.split(os.path.split(item.strm_path)[0])
>
> Por q no usas filetools en lugar de os.path?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/superberny70/pelisalacarta/pull/12#pullrequestreview-6067228>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALOsNVLrNq3vdAB_fvteWffak_FVvv_Gks5q4LIPgaJpZM4KhFTF>
> .
>
